Pod/Classes/OpaqueImageViewComponent/AROpaqueImageView.h

```diff
@@ -4,4 +4,6 @@
 @property (nonatomic, strong, readwrite) NSURL *imageURL;
 @property (nonatomic, strong, readwrite) UIColor *placeholderBackgroundColor;
 @property (nonatomic, readwrite) BOOL *noAnimation;
+@property (nonatomic, readwrite) BOOL *failSilently;
+@property (nonatomic, readwrite) BOOL *highPriority;
```

Comment:
Declaring a scalar property is not typically done as a pointer. I suspect things just happen to work because of `NULL` pointer checking? (I would also annotate them explicitly as `assign`.)

```suggestion
@property (nonatomic, assign, readwrite) BOOL highPriority;
```

(Same applies to the above.)